.github/dependabot.yml

```
@@ -4,3 +4,9 @@ updates:
     directory: /
     schedule:
       interval: weekly
+ignore:
+  # While OTel Android is experimental and has frequent breaking changes, don't upgrade it.
+  - dependency-name: "io.opentelemetry:*"
+  - dependency-name: "io.opentelemetry.android:*"
+  - dependency-name: "io.opentelemetry.contrib:*"
+  - dependency-name: "io.opentelemetry.semconv:*"
```

can we just do this?
```suggestion
  - dependency-name: "io.opentelemetry*"
```

---

i don't know if it works, but we can try it. 🤷‍♀️